applications/sandbox/systems/physics_debugging_system.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <core/core.hpp>
+#include <application/application.hpp>
+#include <physics/events/events.hpp>
+
+namespace lgn = legion;
```

Comment:
this shouldn't be necessary? i think core.hpp already does this? it's fine by me as long as it doesn't give problems tho